.eslintrc

```diff
@@ -61,6 +61,7 @@
       "objectsInObjects": false,
       "arraysInObjects": false
     }],
+    "object-shorthand": [2, "properties"],
```

Comment:
last conversation i think we had in channel, lets go with error. better for consistency. you can leave the change here just make sure to pull i the latest and run the linter again